hamnosys_dicts.py

```diff
@@ -0,0 +1,238 @@
+    "7": "Dom hand - Handshape - Bending",
+    "8": "Dom hand - Handposition - Extended finger direction",
+    "9": "Dom hand - Handposition - Palm orientation",
+    "10": "Dom hand - Handlocation - Frontal plane l/r",
+    "11": "Dom hand - Handlocation - Frontal plane t/b",
+    "12": "Dom hand - Handlocation - Distance from the body",
+}
+
+HandlocationFronalPlaneRL = {
+    "0": "",  # left to the left
+    "1": "",  # left
+    # "2" : "", #center
+    "3": "",  # right
+    "4": "",  # right to the right
+}
+
+HandLocationFronalPlaneTB = {
+    #   "0" : None,
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+    "8": "",
+    "9": "",
+    "10": "",
+    "11": "",
+    "12": "",
+    "13": "",
+    "14": "",
+    "15": "",
+    "16": "",
+    "17": "",
+    "18": "",
+    "19": "",
+    "20": "",
+    "21": "",
+    "22": "",
+    "23": "",
+    "24": "",
+    "25": "",
+    "26": "",
+    "27": "",
+    "28": "",
+    "29": "",
+    "30": "",
+    "31": "",
+    "32": "",
+    "33": "",
+    "34": "",
+    "35": "",
+    "36": "",
+}
+
+HandshapeBaseformsDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+    "8": "",
+    "9": "",
+    "10": "",
+    "11": "",
+}
+
+HandshapeBendingDict = {
+    #   "0" : None,
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+}
+
+HandpositionFingerDirectionDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+    "8": "",
+    "9": "",
+    "10": "",
+    "11": "",
+    "12": "",
+    "13": "",
+    "14": "",
+    "15": "",
+    "16": "",
+    "17": "",
+}
+
+HandpositionPalmOrientationDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+}
+
+HandshapeThumbPositionDict = {
+    #   "0" : None
+    "1": "",
+    "2": "",
+    "3": "",
+}
+
+HandLocationDistanceDict = {
+    "0": "",
+    "1": "",
+    "2": "",
+    #   "3" : None
+    "4": "",
+    "5": "",
+    "6": "",
+}
+
+MovementSigns = {
+    "0": "",
+    "1": "",
+    "2": "",
+    "3": "",
+    "4": "",
+    "5": "",
+    "6": "",
+    "7": "",
+    "8": "",
+    "9": "",
+    "10": "",
+    "11": "",
+    "12": "",
+    "13": "",
+    "14": "",
+    "15": "",
+    "16": "",
+    "17": "",
+    "18": "",
+    "19": "",
+    "20": "",
+    "21": "",
+    "22": "",
+    "23": "",
+    "24": "",
+    "25": "",
+    "26": "",
+    "27": "",
+    "28": "",
+    "29": "",
+    "30": "",
+    "31": "",
+    "32": "",
+    "33": "",
+    "34": "",
+    "35": "",
+    "36": "",
+    "37": "",
+    "38": "",
+    "39": "",
+    "40": "",
+    "41": "",
+    "42": "",
+    "43": "",
+    "44": "",
+    "45": "",
+    # 
```

Comment:
is the comment needed?

Reply:
those are remaining movements signs that were not added to the dict since they do not showup in our datasets, but this shold be added to the dict @MilenaOlech

Reply:
fixed